warplib/src/conversions/integer_conversions.cairo

```diff
@@ -33,3 +36,34 @@ fn bool_into_felt252(val: bool) -> felt252 {
     let resp = OptionTraitImpl::<felt252>::unwrap(resp_option);
     resp
 }
+
+// Cairo-1.0 corelib code from the RC
+impl U256TryIntoFelt252 of TryInto<u256, felt252> {
+    fn try_into(self: u256) -> Option<felt252> {
+        let FELT252_PRIME_HIGH = 0x8000000000000110000000000000000_u128;
+        if self.high > FELT252_PRIME_HIGH {
+            return Option::None(());
+        }
+        if self.high == FELT252_PRIME_HIGH {
+            // since FELT252_PRIME_LOW is 1.
+            if self.low != 0 {
+                return Option::None(());
+            }
+        }
+        Option::Some(
+            self.high.into() * 0x100000000000000000000000000000000_felt252 + self.low.into()
+        )
+    }
+}
+
+fn unsafe_contract_address_from_u256(x: u256) -> ContractAddress {
```

Comment:
can keep this function in the util.cairo file

Reply:
What is the distinction logic? It seems like integer conversion. Well, definitely from integer, and arguably into integer.

Reply:
Sorry again confused with the file https://github.com/NethermindEth/warp/blob/5a0728efab84613fb06d546cab82129db927258d/warplib/src/maths/utils.cairo